include/sound/sof/dai-imx.h

```diff
@@ -51,4 +51,11 @@ struct sof_ipc_dai_sai_params {
 	uint16_t tdm_slot_width;
```

Comment:
Please rephrase the commit message, for this patch, maybe "Add support for configuring PDM interface from topology".
Other than that, LGTM.

Reply:
@iuliana-prodan updated commit message as you suggested.